contrib/babelfishpg_tsql/sql/babelfishpg_tsql.sql

```diff
@@ -3754,3 +3754,152 @@ BEGIN
 END;
 $$ LANGUAGE pltsql;
 GRANT EXECUTE ON PROCEDURE sys.sp_procedure_params_100_managed TO PUBLIC;
+
+CREATE OR REPLACE PROCEDURE sys.sp_helplogins_internal_logins() 
+LANGUAGE pltsql
+AS $$
+BEGIN
+	IF is_srvrolemember('securityadmin') = 0 
+  BEGIN
+    RAISERROR('User does not have permission to perform this action.', 16, 1);
+		RETURN 0;
+  END
+
+	SELECT DISTINCT
```

Comment:
Some possible optimizations:
1. Do we need DISTINCT?
2. Instead of temp table, Can we utilize CTE? Can you please post the plan of both the variation. It will help us decide with one to choose.